x-pack/plugins/secops/public/components/page/overview/translations.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const PACKETBEAT_FLOW = i18n.translate(
+  'xpack.secops.overviewNetwork.source.packetbeatFlowTitle',
+  {
+    defaultMessage: 'Packetbeat Flow',
+  }
+);
+
+export const PACKETBEAT_DNS = i18n.translate(
+  'xpack.secops.overviewNetwork.source.packetbeatDNSTitle',
+  {
+    defaultMessage: 'Packetbeat DNS',
+  }
+);
+
+export const FILEBEAT_SURICATA = i18n.translate(
+  'xpack.secops.overviewNetwork.source.filebeatSuricataTitle',
+  {
+    defaultMessage: 'Filebeat Suricata',
+  }
+);
+
+export const FILEBEAT_ZEEK = i18n.translate(
+  'xpack.secops.overviewNetwork.source.filebeatZeekTitle',
+  {
+    defaultMessage: 'Filebeat Zeek',
+  }
+);
+
+export const AUDITBEAT_SOCKET = i18n.translate(
+  'xpack.secops.overviewNetwork.source.auditbeatSocketTitle',
+  {
+    defaultMessage: 'Auditbeat Socket',
+  }
+);
+
+export const HOSTS_HEADING = i18n.translate('xpack.secops.overview.hosts.heading', {
```

Comment:
Nit: So it looks like in the [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#naming-convention) that headings should end with a `Title` `Type`.